Dockerfile

```diff
@@ -1,12 +1,18 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
-
-# Get the files of the project
+# Copying the repository inside the container
 RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
```

Comment:
Is there a reason to have these `RUN` commands ? If we just `docker build -t introML .` or something similar, then the release shouldn't need to be fetched.

I imagine the likely workflow will be that we push the whole repo to CC and use singularity to build it there. Is there a benefit to grabbing the files separately?

Reply:
My bad I forgot those haha